tests/examples/examples.cfg

```diff
@@ -1,39 +1,29 @@
-[Example-Stehekin-base-case]
+[Example-Classic-Stehekin-base-case]
+driver = classic
 test_description = Short example simulation of the Stehekin basin.
-global_parameter_file = global_param.STEHE.txt
+global_parameter_file = global_param.classic.STEHE.txt
 check = complete, nonans
 
-[Example-Stehekin-3hewb]
-test_description = Short example simulation of the Stehekin basin using 3hr engergy balance.
-global_parameter_file = global_param.STEHE.3hewb.txt
+[Example-Classic-Stehekin-fewb]
+driver = classic
+test_description = Short example simulation of the Stehekin basin using 1hr full energy balance.
+global_parameter_file = global_param.classic.STEHE.fewb.txt
```

Comment:
`fewb` is a confusing indicator for the global file since I would take that to read as "full energy water balance", which are mutually exclusive options in VIC (even though the full energy also calculates a water balance, that is not how the terminology is used in most of VIC)